integration-tests/docker/broker.conf

```diff
@@ -21,7 +21,7 @@ command=java
   -Ddruid.broker.cache.populateCache=true
   -Ddruid.cache.type=local
   -Ddruid.cache.sizeInBytes=40000000
-  -cp /usr/local/druid/lib/*
+  -cp /shared/docker/lib/*
```

Comment:
not sure why we have this change? thought libs are always `/usr/local/druid/lib/*`?

Reply:
It makes it easier to recompile libs without rebuilding the image, when you fixing/debugging tests locally. For example, if you re-run tests multiple times locally then rebuilding the image is not required and time-consuming.